.github/workflows/prune-tags.yml

```
@@ -18,8 +18,10 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
+
       - name: Gather tags
         id: gather_tags
-        run: .github/list-prune-tags.sh -n ${{ github.ref_name }} >> $GITHUB_OUTPUT
+        run: echo "TAGS=$(.github/list-prune-tags.sh -n ${{ github.ref_name }})" >> $GITHUB_OUTPUT
+
       - name: Prune tags
-        run: git push origin --delete ${{ steps.gather_tags.output }}
+        run: git push origin --delete ${{ steps.gather_tags.output.TAGS }}
```

Based on the workflow commands doc you linked to:

```suggestion
        run: git push origin --delete ${{ steps.gather_tags.outputs.TAGS }}
```

---

thank you for the catch!